plugins/idea-not-null/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins{
```

Comment:
The root build.gradle should maybe contain `allprojects` and `subprojects` blocks to remove a lot of the redundant setup where it doesn't change, such as plugins, group, compatibility, etc., especially since that means fewer changes are required for updates